contentcuration/contentcuration/frontend/shared/vuex/file/utils.js

```diff
@@ -149,41 +149,37 @@ async function getManifestMetadata(manifestFile, zip, procssedFiles) {
       if (xmlDoc.getElementsByTagName('lomes:title').length) {
         metadata.title = xmlDoc
           .getElementsByTagName('lomes:title')[0]
-          .children[0].textContent.replace(/ {2}|\r\n|\n|\r/gm, '');
+          .children[0].textContent.trim();
       }
       if (
         xmlDoc.getElementsByTagName('lomes:idiom').length &&
-        xmlDoc
-          .getElementsByTagName('lomes:idiom')[0]
-          .textContent.replace(/ {2}|\r\n|\n|\r/gm, '') !== 'und'
+        LanguagesMap.has(xmlDoc.getElementsByTagName('lomes:idiom')[0].textContent.trim()) &&
```

Comment:
Could avoid a bit of repetition here and define outside of the if statement:
```
const language = xmlDoc.getElementsByTagName('lomes:idiom').length ? xmlDoc.getElementsByTagName('lomes:idiom')[0].textContent.trim() : 'und';
```
(defaulting to the disallowed `und`)

then you can do the checks and assignment against this value instead.